start-building/tutorials/how-to-use-the-sdk/connect-to-the-network.md

```diff
@@ -0,0 +1,44 @@
+---
+description: How to connect with to VechainThor blockchain using @vechain/sdk-network
+---
+
+# Connect to the Network
+
+Interacting with VechainThor requires only an instance of `ThorClient` that is configured for the relevant network. Once the connection is established, this instance can be utilized to interact in an asynchronous manner. Below is a snippet that demonstrates importing the library and initializing a client:
+
+```js
+import { ThorClient } from "@vechain/sdk-network";
+const thor = ThorClient.fromUrl("https://mainnet.vechain.org");
+```
+
+The snippet connects to the MainNet, where all production related activity is found.
+
+Additionally there is a TestNet available for testing and development purposes. This allows developers to experiment without risking real assets.
+
+To connect to the TestNet use `https://testnet.vechain.org` as URL. If you have deployed a local solo instance of Vechain, it is normally available on `http://localhost:8669`.
```

Comment:
Can you give more details about Thor solo node integration?
Basically we offer a Thor-solo node seeded with some account in order to have a great test environment (feel free to ask if any info)

Reply:
@rodolfopietro97 I was looking for some type documentation but was unable to find them in the current docs (was checking on https://docs.vechain.org/developer-resources/sdks-and-providers/sdk/thor-client#blocks and the sdk repo) but not was thinking about tsdocs.

Do you suggest these links?

- https://tsdocs.dev/docs/@vechain/sdk-network/1.0.0-beta.2/interfaces/network.CompressedBlockDetail.html
- https://tsdocs.dev/docs/@vechain/sdk-network/1.0.0-beta.2/interfaces/network.BlockDetail.html

Thanks a lot for the hint! I missed this too.

Reply:
Regarding solo node, I would refer to the existing documentation on https://docs.vechain.org/start-building/tutorials/how-to-run-a-thor-solo-node – do you think this is sufficient?  

I believe it describes it already in a wonderful easy way. Will also unify the wording into "solo node", just seen that I used "local solo instance" there.

Reply:
@ifavo For Thor solo node you can refers to official doc or document our Thor solo solution.
Basically if you do `yarn start-thor-solo` you will start a solo node with some seeded accounts

For types yes, you can use typedoc link as reference

Reply:
> @ifavo For Thor solo node you can refers to official doc or document our Thor solo solution. Basically if you do `yarn start-thor-solo` you will start a solo node with some seeded accounts
> 
> For types yes, you can use typedoc link as reference

The solo node reference is a side note, I prefer to keep this page focused on how to create your thor client, so the link to the other doc page should be sufficient.

Knowing that it can be started as a docker container within the SDK is good to know to! Thank you!